starfish/pipeline/features/spots/detector/detect.py

```diff
@@ -0,0 +1,222 @@
+from functools import partial
+from itertools import product
+from typing import Callable, Dict, Sequence, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from starfish.constants import Indices, Features
+from starfish.image import ImageStack
+from starfish.intensity_table import IntensityTable
+from starfish.munge import dataframe_to_multiindex
+from starfish.pipeline.features.spot_attributes import SpotAttributes
+from starfish.typing import Number
+
+
+def measure_spot_intensity(
```

Comment:
This has been moved from `GaussianSpotDetector` but aspects (lines 48-52) were moved from the spot detector into this specific method because (1) the bounding box (zmin/max, ymin/max, xmin/max) are not used outside this function, and (2) they're needed for the trackpy-locate based method, so this is the best place for that code.